tapfreighter/chain_porter.go

```diff
@@ -132,7 +132,7 @@ func NewChainPorter(cfg *ChainPorterConfig) *ChainPorter {
 	)
 	return &ChainPorter{
 		cfg:             cfg,
-		outboundParcels: make(chan Parcel),
+		outboundParcels: make(chan Parcel, 10),
```

Comment:
I think we want to leave this as unbuffered? Otherwise we would end up trying to start the state machine with multiple packets at once:

https://github.com/lightninglabs/taproot-assets/blob/1b5a4ef392dcad4906272b1ef805d7333f5d4080/tapfreighter/chain_porter.go#L318